addons/website/controllers/main.py

```diff
@@ -692,32 +692,32 @@ def google_maps_api_key(self):
     # Themes
     # ------------------------------------------------------
 
-    def _get_customize_views(self, xml_ids):
```

Comment:
I stopped my review of this PR here: you are in a stable version, you cannot rename/remove functions. What you could do is create the new _get_customize_data function, make the old _get_customize_views call it and add a comment to indicate it should be removed.